docs/navigation-parameters-reference.md

```diff
@@ -1,269 +1,293 @@
-# Navigation parameters reference
-You can use the listed parameters and functions to configure your navigation structure. The examples show how to use selected options.
+# Navigation properties reference
 
-## Navigation configuration example
+> NOTE: To see the navigation properties in use, check out the [navigation configuration example](navigation-configuration-example.md) or configure a test application in the [Luigi fiddle](https://fiddle.luigi-project.io). 
 
-This code sample demonstrates a sample structure with the parameters you can use when configuring navigation for Luigi.
+You can use properties and functions in this reference to configure your Luigi navigation structure.
 
+* [Routing properties](#routing-properties)
+* [Navigation properties](#navigation-properties)
+* [Node properties](#node-properties)
+* [Profile](#profile) 
+* [Context switcher](#context-switcher)
+* [Product switcher](#product-switcher)
+* [App switcher](#app-switcher)
+
+
+## Routing properties
+You can configure the way Luigi tackles routing in your application in the `routing:` section of the configuration file. For example, you can choose the routing strategy to apply in your application as either hash or path location routing.
+
+### useHashRouting
+- **type**: boolean 
+- **required**: yes 
```

Comment:
Just to clarify, is the default hash- or path-based routing? Perhaps this information would be useful to include (maybe in place of required since Philipp said nothing is required...)